contracts/infrastructure/ens/ArgentENSResolver.sol

```diff
@@ -24,7 +24,7 @@ import "./ENSResolver.sol";
  * to the list of resolved names.
  * @author Julien Niset - <julien@argent.im>
  */
-contract ArgentENSResolver is Owned, Managed, ENSResolver {
+contract ArgentENSResolver is Owned, Managed, IENSResolver {
```

Comment:
Is there any value in updating the ENSResolver other than changing the solidity version and "cleaning" our folder structure? Not sure it is compelling enough to add a second instance of a perfectly working contract.

Reply:
We don't have to redeploy it. As you said, just cleaning up the contracts.